packages/mariadb/src/MariaDbPlatform.ts

```diff
@@ -74,4 +74,21 @@ export class MariaDbPlatform extends AbstractSqlPlatform {
     return `alter table ${quotedTableName} add fulltext index ${quotedIndexName}(${quotedColumnNames.join(',')})`;
   }
 
+  private readonly NULLS_TRANSLATE = {
+    [QueryOrder.asc_nulls_first]: 'is not null',
+    [QueryOrder.asc_nulls_last]: 'is null',
+    [QueryOrder.desc_nulls_first]: 'is not null',
+    [QueryOrder.desc_nulls_last]: 'is null',
+  } as const;
+
+  override getOrderByExpression(column: string, direction: string): string[] {
```

Comment:
i guess we dont need to test the mariadb explicitly, lets just add `/* istanbul ignore next */` here
```suggestion
  /* istanbul ignore next */
  override getOrderByExpression(column: string, direction: string): string[] {
```